get_started/index.md

```diff
@@ -7,21 +7,23 @@ sidebar:
 
 On behalf of the developers, contributors and user community: **welcome to ROOT!**
 
-The [ROOT beginners' guide (aka "Primer")]({{ '/primer' | relative_url }})
-is certainly the first document to read to master the power of ROOT.
+If you have never used ROOT before and don't know where to start, we recommend that you first explore: https://github.com/root-project/student-course
 
-The [Manual]({{ '/manual' | relative_url }}) introduces you to the main parts of ROOT.
+For that purpose you don't need to install ROOT on your machine, you can directly run all the examples and exercises 
+on [SWAN](https://swan.cern.ch){:target="_blank"} (if you have a CERN computing account), otherwise either using github codespaces or binder. 
```

Comment:
```suggestion
on [SWAN](https://swan.cern.ch){:target="_blank"} (if you have a CERN computing account), or otherwise using GitHub Codespaces or Binder. 
```